process/sync/shardblock.go

```diff
@@ -26,13 +25,7 @@ import (
 type ShardBootstrap struct {
 	*baseBootstrap
 
-	miniBlocks storage.Cacher
-
-	chRcvMiniBlocks  chan bool
-	mutRcvMiniBlocks sync.Mutex
-
-	resolversFinder    dataRetriever.ResolversFinder
-	miniBlocksResolver dataRetriever.MiniBlocksResolver
+	resolversFinder dataRetriever.ResolversFinder
```

Comment:
You could remove resolverFinder from ShardBootstrap struct, is unused in ShardBootstrap class beside New... method

Reply:
done